src/System.Net.Security/tests/UnitTests/System/Security/Authentication/ExtendedProtection/ExtendedProtectionPolicyTest.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Security.Authentication.ExtendedProtection;
+
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    public class ExtendedProtectionPolicyTest
+    {
+        [Fact]
+        public void Constructor_PolicyEnforcement_NeverParam()
+        {
+            var paramName = "policyEnforcement";
+            var ex = Assert.Throws<ArgumentException>(() => new ExtendedProtectionPolicy(PolicyEnforcement.Never, ProtectionScenario.TransportSelected, null));
+            Assert.Equal(typeof(ArgumentException), ex.GetType());
+            Assert.Null(ex.InnerException);
+            Assert.NotNull(ex.Message);
+            Assert.NotNull(ex.ParamName);
+            Assert.Equal(paramName, ex.ParamName);
```

Comment:
All of this stuff is unnecessary. You could just do:
`Assert.Throws<ArgumentException>("policyEnforcement", () => ...);`